pom.xml

```diff
@@ -28,8 +28,8 @@ THE SOFTWARE.
    <parent>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>plugin</artifactId>
-      <version>1.424.6</version>
-      <relativePath />
+      <version>1.609.3</version>
```

Comment:
For now I will leave as is and if in the future we need to move forward, let's open an issue. Changing parent pom could generate any incompatibility? For what I see is that it is only going to work in Jenkins 2?